applications/FemToDemApplication/custom_python/fem_to_dem_python_application.cpp

```diff
@@ -61,6 +61,10 @@ namespace Python
     AddCustomProcessesToPython(m);
 
     //registering variables in python
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m,ACCELERATION_BACKUP);
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m,ACCELERATION_PREVIOUS_BACKUP);
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m,DISPLACEMENT_BACKUP);
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m,DISPLACEMENT_PREVIOUS_BACKUP);

```

Comment:
Remove this please and use the buffer from DISPLACEMENT_BACKUP, idem for ACCELERATION_BACKUP